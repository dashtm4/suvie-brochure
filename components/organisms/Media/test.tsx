import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Media } from '.'

describe('<Media />', () => {
  const imageSrc = 'image.jpg'

  it('should renders correct <Media />', () => {
    const wrapper = shallow(<Media imageSrc={imageSrc} />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
