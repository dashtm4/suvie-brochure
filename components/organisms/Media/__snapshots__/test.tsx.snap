// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Media /> should renders correct <Media /> 1`] = `
<figure
  className="relative overflow-hidden md:mr-auto"
>
  <VideoPlayer
    autoplay={true}
    className="absolute inset-0 object-cover w-full h-full"
    controls={false}
    loop={true}
    muted={true}
    playsInline={true}
    poster="image.jpg"
  />
</figure>
`;
